exercises/scrabble-score/tests/scrabble-score.rs

```diff
@@ -67,3 +67,10 @@ fn empty_words_are_worth_zero() {
 fn all_letters_work() {
     assert_eq!(score("abcdefghijklmnopqrstuvwxyz"), 87);
 }
+
+#[test]
+#[ignore]
+fn german_letters_do_not_score() {
+    assert_eq!(score("STRASSE"), 7, "\"SS\" should score 2.");
```

Comment:
inconsistency: In all other lines added/modified in this PR, there was no trailing period, unlike this line which does.